<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainFile1</class>
 <widget class="QMainWindow" name="MainFile1">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainFile1</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="firstno">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>First No. :-</string>
    </property>
   </widget>
   <widget class="QLabel" name="secondno">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Second No. :-</string>
    </property>
   </widget>
   <widget class="QLabel" name="result">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>RESULT :-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMinus">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMultiply">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDivide">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>210</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtFirst">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>70</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtSecond">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtResult">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>150</y>
      <width>251</width>
      <height>24</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
